### PR TITLE
[ DO NOT MERGE] 2.0 branch for eZ Platform 3.0 / Sf 4 compatibility

### PR DESCRIPTION
**Pull request by bdunogier**
Future 2.0 branch/version of the package.

### Changes
- renamed .yml to .yaml
- updated README
- updated requirements for ezplatform 3.0
- updated branch alias to 2.0.x-dev

### TODO
- [ ] repository-forms / admin-ui upgrade
- [x] `types` sub-directory for GraphQL types
- [ ] Installation instructions & flex

### CODE REVIEW

**Comment by bdunogier** [comment]
Great, thank you for the feedback @adamwojs. I'll take care of this soon.

**Comment by bdunogier** [comment]
All taken care of, as far as I can tell.

File: src/Symfony/DependencyInjection/EzSystemsEzPlatformQueryFieldTypeExtension.php [hunk 1]
```diff
@@ -80,24 +80,30 @@ private function prependJMSTranslationConfig(ContainerBuilder $container): void
...
     protected function prependFieldTemplateConfig(ContainerBuilder $container): void
     {
-        $configFile = __DIR__ . '/../Resources/config/field_templates.yml';
+        $configFile = __DIR__ . '/../Resources/config/field_templates.yaml';
         $config = Yaml::parse(file_get_contents($configFile));
         $container->prependExtensionConfig('ezpublish', $config);
         $container->addResource(new FileResource($configFile));
     }
+
+    private function prependGraphQL(ContainerBuilder $container)
```
Comment (adamwojs) [inline src/Symfony/DependencyInjection/EzSystemsEzPlatformQueryFieldTypeExtension.php]:
```suggestion
    private function prependGraphQL(ContainerBuilder $container): void
```

**Comment by bdunogier** [comment]
> I found the following issues, besides inline comments:
All taken care of.

**Comment by bdunogier** [comment]
Not merging, since it isn't meant to be. Welcome to life, 2.0 :)

**Pull request closed without merging**